snap/snapcraft.yaml

```diff
@@ -34,13 +34,15 @@ parts:
     plugin: go
     source: .
     build-snaps:
-      - go/1.21/stable
+      - go/1.23/stable
```

Comment:
Let's try to only include changes which are strictly relevant to the PR.

Reply:
Reverted in https://github.com/letFunny/chisel/pull/13/commits/6e14e4e82a1a2fa8f7164d16d7f4b74617cc8b8a. Thanks.